src/adhocracy_core/docs/deletion.rst

```diff
@@ -1,123 +1,123 @@
-  backreferences, which are calculated on demand. The
-  *include=deleted|hidden|all* parameter can be used to change that and
-  include backreferences to deleted and/or hidden resources. The same
-  restrictions apply, i.e. normal users can use this parameter to find out
-  whether hidden backreferences exist, but they won't be able to see their
-  contents. In any case the frontend should be prepared to deal with
-  *410 Gone* when following backreferences in the same way as when
-  following forward reference -- even if it didn't explicitly ask to include
-  them, they might show up due to caching.
+  *Hidden* resources will normally not be shown in backreferences, which
+  are calculated on demand. The *include=hidden* parameter can be used
+  to change that and include backreferences to hidden resources. The
+  same restrictions apply, i.e. normal users can use this parameter to
+  find out whether hidden backreferences exist, but they won't be able
+  to see their contents. In any case the frontend should be prepared to
+  deal with *410 Gone* when following backreferences in the same way as
+  when following forward reference -- even if it didn't explicitly ask
+  to include them, they might show up due to caching.
 
 FIXME We should extend the Meta API to expose the distinction between
 references and backreferences to the frontend, currently only the backend
 knows this.
 
 Notes:
 
-* Physical deletion of resources (pruning them from the DB without hope of
-  resurrection) is not currently possible through the REST API. There may be
-  server-side processes that prune some specific or even all *hidden*
-  resources from the DB from time to time or when started by an admin, but
-  they cannot be controlled via REST and are not specified or documented
-  here.
-* The HTTP DELETE command is not used since it would imply physically
-  deleting a resource.
-* This document is about deletion of resources (JSON documents). Deletion
-  of uploaded assets (images, PDFs etc.) is outside its current scope.
-* Currently, the deleted/hidden status of resources isn't treated as special
-  by the Websocket server. So, if an resource is flagged as deleted/hidden
-  (or undeleted etc.), a "modified" event is sent to subscribers of that
-  resource and a "modified_child" event is sent to subscribers of the parent
-  pool. FIXME At same point in the future, we might want to change that and
+* This document is about deletion of resources (JSON documents).
+  Deletion of uploaded assets (images, PDFs etc.) is outside its current
+  scope.
+* Currently, the hidden status of resources isn't treated as special by
+  the Websocket server. So, if an resource is flagged as hidden, a
+  "modified" event is sent to subscribers of that resource and a
+  "modified_child" event is sent to subscribers of the parent pool.
+  FIXME At same point in the future, we might want to change that and
   send "removed"/"removed_item" messages instead.
```

Comment:
cleanup not related to this pull request: can be removed to cleanup fixmes we won't fix or are part of the refactoring list